content/en/docs/concepts/architecture/controller.md

```diff
@@ -166,5 +166,5 @@ controller does.
 * Learn more about the [Kubernetes API](/docs/concepts/overview/kubernetes-api/)
 * If you want to write your own controller, see
   [Extension Patterns](/docs/concepts/extend-kubernetes/#extension-patterns)
-  in Extending Kubernetes.
+  in Extending Kubernetes and [sample-controller](https://github.com/kubernetes/sample-controller) repository.
```

Comment:
maybe we can remove "in Extending Kubernetes" safely?
It doesn't add any value and the uppercase "Extending" looks weird.

Reply:
"and" -> "and the" ?

Reply:
Only writing Extension patterns also looks incomplete. How about this:
`Learn how to write your own controller with 
  [Kubernetes extension patterns](/docs/concepts/extend-kubernetes/#extension-patterns)  
 and the [sample-controller](https://github.com/kubernetes/sample-controller) repository.`

Reply:
Updated. Thanks. @Ritikaa96  @tengqm